files/rsyslog

```diff
@@ -2,9 +2,9 @@ RSYSLOGCUSTOMHEADER (?:(?:<%{NONNEGINT:syslog_abspri}>(?:%{NONNEGINT:syslog_vers
 
 # new style, needs custom logstash config
 RSYSLOG_SSHD %{RSYSLOGCUSTOMHEADER} %{IPORHOST:syslog_hostname} (?<program>sshd)(?:\[%{POSINT:pid}\])?: %{SSH_MESSAGE}
-RSYSLOG_MSG (?:%{MODULECMD_MSG}|%{NFS_MSG}|%{CEPH_MSG}|%{GREEDYDATA:syslog_message})
+RSYSLOG_MSG (?:%{MODULECMD_MSG}|%{NFS_MSG}|%{CEPH_MSG}|%{OPENNEBULA_MSG}|%{KEYVALUE_MSG}|%{GREEDYDATA:syslog_message})
```

Comment:
can you add a test for some non-matching input data and make sure it ends up in `syslog_message`? (it wouldn't have mattered in this case, but it seems missing)